R/textstat_simil.R

```diff
@@ -344,11 +364,12 @@ as_dist <- function(x, method, call, diag = FALSE, upper = FALSE) {
 #' @export
 #' @method as.matrix simil
 #' @keywords textstat internal
-as.matrix.simil <- function(x, diag = 1.0, ...) {
+as.matrix.simil <- function(x, diag = 1.0, upper = TRUE, ...) {
```

Comment:
We should also return any values thresholded out from `min_simil` as `NA` in the matrix. The 0's below should be `NA`, not 0.
```r
> textstat_simil(dfmat, method = "cosine", margin = "documents", tri = FALSE, min_simil = 0.50) %>%
+     as.matrix()
           2001-Bush 2005-Bush 2009-Obama 2013-Obama 2017-Trump
2001-Bush  1.0000000 0.5204355  0.5411649  0.5561972          0
2005-Bush  0.5204355 1.0000000  0.0000000  0.5163644          0
2009-Obama 0.5411649 0.0000000  1.0000000  0.6373318          0
2013-Obama 0.5561972 0.5163644  0.6373318  1.0000000          0
2017-Trump 0.0000000 0.0000000  0.0000000  0.0000000          1
```

Reply:
Need to find an efficient way to fill up a matrix with NA.

Reply:
Yes I wish there were such a way! But non-recorded values can only have one value, otherwise they'd need to be recorded.

Reply:
Fixed in d94cbd7d.